client/packages/requisitions/src/ResponseRequisition/DetailView/IndicatorsTab.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  Box,
+  ButtonWithIcon,
+  IndicatorLineRowNode,
+  NothingHere,
+  PlusCircleIcon,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { ProgramIndicatorFragment, ResponseFragment } from '../api';
+
+interface IndicatorTabProps {
+  onClick: (
+    programIndicator?: ProgramIndicatorFragment,
+    indicatorLine?: IndicatorLineRowNode,
+    response?: ResponseFragment
+  ) => void;
+  isLoading: boolean;
+  response?: ResponseFragment;
+  indicators?: ProgramIndicatorFragment[];
+}
+
+export const IndicatorsTab = ({
+  onClick,
+  isLoading,
+  response,
+  indicators,
+}: IndicatorTabProps) => {
+  const t = useTranslation();
+  if (isLoading) {
+    return <NothingHere body="There are no indicators for this requisition" />;
+  }
+  const hivIndicators = indicators?.filter(
+    indicator => indicator?.code === 'HIV'
+  );
+  const regimenIndicators = indicators?.filter(
+    indicator => indicator?.code === 'REGIMEN'
+  );
```

Comment:
Seems like it might be easiest just to hide/show these buttons based on if any indicators of that kind are defined? Though I suppose then what do you show if neither are available... snack seems nice 😁

Reply:
Wanna filter this by program ID too... to get the correct indicators >.>

Reply:
I think it's better to hide the buttons as the niche of users trained on this feature should know the deal and if the button isn't there it saves them the effort of clicking to find out it's not there. We already hide the tab entirely if there are no indicators for the given requisition/program/period. HIV indicators for example, being present should really be self evident based on it being an HIV program to begin with.

Ideally to match the configurable schema these buttons (or tables in the future) would be dynamically generated rather than hardcoded, in which case you'd effectively/hide show based on existence too.